shop/models/product.py

```diff
@@ -135,21 +135,27 @@ def get_product_variant(self, **kwargs):
 
     def get_availability(self, request):
         """
-        Hook for checking the availability of a product. It returns a list of tuples with this
-        notation:
-        - Number of items available for this product until the specified period expires.
-          If this value is ``True``, then infinitely many items are available.
-        - Until which timestamp, in UTC, the specified number of items are available.
-        This function can return more than one tuple. If the list is empty, then the product is
-        considered as not available.
-        Use the `request` object to vary the availability according to the logged in user,
-        its country code or language.
+        Hook for checking the availability of a product.
+
+        Args:
+            Use the ``request`` object to vary the availability according to the logged
+            in user, its geo-location, language or whatever can be determined using the 
+            request object. 
```

Comment:
"whatever can be determined using the request object" -> "anything else that depends on the request"